specification/attributes/null_handling.md

```diff
@@ -20,7 +20,8 @@ Indicates how to handle columns that don't have a value.
 ## Requirements
 
 * Columns MUST use NULL when there isn't a value that can be specified for a nullable column.
-* Columns MUST NOT use empty strings or placeholder values such as "Not Set" or "Not Applicable" in columns, regardless of if the column allows nulls or not.
+* String columns MUST NOT use empty strings or placeholder values such as "Not Set" or "Not Applicable" in columns, regardless of whether the column allows nulls or not.
+* Numeric columns MUST use NULL and MUST NOT use 0 when there isn't a value, regardless of whether the column allows nulls or not.
```

Comment:
I think the 'regardless of whether the column allows null or not' won't work if we're proposing people use null. If it's not allowing null, a valid value will have to be provided and often, the valid value will be 0 (e.g. cost columns). If the column allows NULL, then we don't want people using 0s when the column is not applicable.

Reply:
This also means rows like tax would have a unit price of $0 and quantity of 0. Which seems like a bad idea (and also breaks P*Q.

Reply:
That was there before so I just left it. I believe the point is that, even if the spec "requires" a value, there may be a scenario where a provider can't supply it. And in that case, they MUST NOT use a placeholder. At least that's how I read it. The line I added about numbers was to ensure they don't use `0` in that case, since it would break validation scenarios. If there's no value, it needs to be null to convey that there's no value.